### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by kbart93**

### CODE REVIEW

File: test_api_crud.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import requests
+import json
+
+def test_get_user():
+    p = {"page": 2}
+    resp = requests.get("https://reqres.in/api/users", params=p)
+    json_response = resp.json()
+
+    print(json_response["total_pages"])
```
Comment (przemekpln92) [inline test_api_crud.py]: it is better to use logger for info messages ex. logging.info(json_response["total_pages"]), print is not a good practice

File: test_api_crud.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        "name": "Jan",
+        "job": "Automation"
+    }
+
+    resp = requests.post("https://reqres.in/api/users", data=payload)
+    print(resp.json())
+    assert resp.json()["job"] == "Automation", "wrong data!"
+    assert resp.json()["name"] == "Jan", "wrong data!"
+
+    data2 = {
```
Comment (przemekpln92) [inline test_api_crud.py]: try to give variable more meaningful names data2 is not especially descriptive, also test should follow AAA convention - Arange Act Assert so keep payloads at the beginning

**Review by przemekpln92 — CHANGES_REQUESTED** [review]

File: test_api_crud.py (hunk 1 above)
Comment (kbart93) [reply test_api_crud.py]: done

File: test_api_crud.py (hunk 2 above)
Comment (kbart93) [reply test_api_crud.py]: I have applied changes

**Pull request merged**